worlds/apsudoku/docs/setup_en.md

```diff
@@ -13,25 +11,33 @@ Does not need to be added at the start of a seed, as it does not create any slot
 
 ## Installation Procedures
 
-Go to the latest release from the [APSudoku Releases page](https://github.com/EmilyV99/APSudoku/releases). Download and extract the `APSudoku.zip` file.
+Go to the latest release from the [APSudoku Releases page](https://github.com/APSudoku/APSudoku/releases/latest). Download and extract the appropriate file for your platform.
 
 ## Joining a MultiWorld Game
 
-1. Run APSudoku.exe
-2. Under the 'Archipelago' tab at the top-right:
-	- Enter the server url & port number
+1. Run the APSudoku executable.
+2. Under `Settings` &rarr; `Connection` at the top-right:
+	- Enter the server IP & port number
```

Comment:
I'm confused, `archipelago.gg` isn't an IP? Maybe you want "address"?

Reply:
. . . this is technically true, though 99% of the time I hear anyone referring to this they "enter the IP and port". So one of those situations where while it isn't *technically* correct, it's also the general plain language used by most people.

Reply:
When you say "most people", I think, at best, that's most people who are already experienced with Archipelago, which is not most people that would be reading the setup guide. Most people reading the setup guide would not be as familiar with common wording used in the AP community, and to them, I think "address" would be less confusing.

Reply:
You know, that is an entirely fair point.